lib/iris/mesh/components.py

```diff
@@ -2808,6 +2759,31 @@ def location(self):
     def axis(self):
         return self._metadata_manager.axis
 
+    @property
+    def points(self):
+        """The coordinate points values as a NumPy array."""
+        if self.timestamp < self.mesh.timestamp or self.timestamp is None:
+            self._values, _, _ = self._load_points_and_bounds()
+        return super._values
+
+    @points.setter
+    def points(self, value):
+        if len(value) > 0:
+            msg = "Cannot set 'points' on a MeshCoord."
+            raise ValueError(msg)
+
+    @property
+    def bounds(self):
+        if self.timestamp < self._mesh.timestamp or self.timestamp is None:
+            _, self.bounds, _ = self._load_points_and_bounds()
+        return super.bounds
+
+    @bounds.setter
+    def bounds(self, value):
+        if len(value) > 0 and self.bounds:
+            msg = "Cannot set 'bounds' on a MeshCoord."
+            raise ValueError(msg)
```

Comment:
These two together create some kind of infinite loop, since the setter accesses the getter then the getter accesses the setter...

Also when I tested this, asking for `my_coord.bounds` returns `None`, but perhaps that problem will go away once the above problem is fixed.